lambdas/get-addresses/src/app.ts

```diff
@@ -1,42 +1,31 @@
 import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
 import { AddressService } from "./services/address-service";
 import { DynamoDbClient } from "./lib/dynamo-db-client";
-import { SsmClient } from "./lib/param-store-client";
-import { ConfigService } from "./services/config-service";
+import { getParameter } from "@aws-lambda-powertools/parameters/ssm";
+import { Logger } from "@aws-lambda-powertools/logger";
 
-const configService = new ConfigService(SsmClient);
-const initPromise = configService.init();
+const logger = new Logger();
+const PARAMETER_PREFIX = process.env.AWS_STACK_NAME || "";
+
+const apiResponse = (statusCode: number, body: string): APIGatewayProxyResult => ({
+    statusCode,
+    body,
+});
```

Comment:
```suggestion
```
This is useful but should be defined separately in a utils file. But for this function we probably don't need it anyway.